src/ImageSharp/Image/ImageBase{TPixel}.cs

```diff
@@ -37,6 +38,11 @@ public abstract class ImageBase<TPixel> : IImageBase<TPixel>
         private TPixel[] pixelBuffer;
 
         /// <summary>
+        /// The span representing the pixel buffer
+        /// </summary>
+        private Span<TPixel> span;
```

Comment:
Assigning it to an auto property would be the same as having it as a field, because autoprops have implicit backing fields. Spans have to be always created on the fly.